packages/api/src/errors/metriport-error.ts

```diff
@@ -8,7 +8,7 @@ export default class MetriportError extends Error {
   constructor(
     message: string,
     cause?: unknown,
-    readonly additionalInfo?: Record<string, string | number | undefined | null>
+    readonly additionalInfo?: Record<string, string | number | boolean | undefined | null>
```

Comment:
For later: also update other instances of MetriportError

Reply:
Will handle in v1.

Reply:
https://github.com/metriport/metriport/pull/2872